plugins/modules/vultr_block_storage.py

```diff
@@ -37,8 +38,20 @@
     description:
       - State of the block storage volume.
     default: present
-    choices: [ present, absent ]
+    choices: [ present, absent, attached, detached ]
     type: str
+  attached_to_SUBID:
+    description:
+      - The ID of the server the volume is attached to.
+      - Required if I(state) is attached.
+    aliases: [ attached_to_id ]
+    type: int
+  live_attachment:
+    description:
+      - Whether the volume should be attached/detached, even if the server not stopped.
+    type: str
```

Comment:
please change this to `bool` (and change the default and choices to be booleans)

Reply:
fixed